src/test/java/com/final_assignment/UserServiceTest.java

```diff
@@ -23,28 +28,76 @@ class UserServiceTest {
     private UserService userService;
 
     @Mock
-    private UserService userMapper;
+    private UserMapper userMapper;
+
+    private List<User> mockUsers;
+
+    @BeforeEach
+    void setUp() {
+        List<User> mockUsers = Arrays.asList(
+                new User(1, "yamada", "yamada@example.com"),
+                new User(1, "kobashi", "kobashi@example.com"),
+                new User(1, "akiyama", "akiyama@example.com"),
+                new User(1, "misawa", "misawa@example.com"),
+                new User(1, "takayama", "takayama@example.com")
+                );
+    }
     
     @Test
-    void findByNameStartingWith() {
+    void 全レコードを取得する事() {
+        doReturn(mockUsers).when(userMapper).findAll();
+        List<User> actual = userService.findByNameStartingWith(null);
+        assertThat(actual).isEqualTo(mockUsers);
+    }
+
+    @Test
+    void クエリ文字列を指定してレコードを取得できること() {
```

Comment:
```suggestion
    void 指定した名前から始まるレコードを取得できること() {
```
クエリ文字列というのはHTTPリクエストの言葉なので業務処理をになうServiceでは使わない方が良いかと。